src/main/java/seedu/badmaths/HelpManual.java

```diff
@@ -1,3 +1,8 @@
+/**
+ * Provides the HelpManual to guide users who are operating BadMaths.
+ *
+ */
+
```

Comment:
It would be better to delete the empty line.

Reply:
If you are meaning line 3 the empty line, it's because I see the example sample they always leave an extra line perhaps for indenting purposes? :O @ZiqiuZeng

Reply:
I think that based on the premise that there are contents next. Since it is single line, there is not necessary to add the empty line 3. @WilsonLee2000